shell/app/common/components/contractive-filter.tsx

```diff
@@ -347,6 +349,8 @@ export const ContractiveFilter = ({ initValue, values, conditions: propsConditio
   const inputList = conditions.filter(a => a.type === 'input');
   const displayConditionsLen = conditions.filter(item => !item.fixed && item.type !== 'input').length;
 
+  if (!visible) return null;
```

Comment:
'visible' should not defined here, this is a pure common component, it only used in config-page